lib/sejfguru/assets/assets.ex

```diff
@@ -32,9 +32,26 @@ defmodule Sejfguru.Assets do
   """
   def list_assets(type: type, page: page) do
     Asset
-      |> Ecto.Query.where(type_name: ^type)
-      |> Ecto.Query.order_by(:name)
-      |> Repo.paginate(page: page)
+    |> Ecto.Query.where(type_name: ^type)
+    |> Ecto.Query.order_by(:name)
+    |> Repo.paginate(page: page)
+  end
+
+   @doc """
+  Returns the list of assets of given type paginated.
+
+  ## Examples
+
+      iex> list_assets(type: "Mobile", page: 1)
+      [%Asset{}, ...]
+
+  """
+  def list_assets_with_users(type: type, page: page) do
```

Comment:
would be nice to have it done more generic - for example function with `preloaded: preloaded` parameter, then it can be used in controller with needed associations as the parameter
but it's good for now